src/updateImageHashes.test.ts

```diff
@@ -1,83 +1,126 @@
 import * as F from "./figma-json";
 import updateImageHashes from "./updateImageHashes";
 
-// 2x3 identitiy matrix
+// 2x3 identity matrix
 const transformDefault: F.Transform = [
   [1, 0, 0],
   [0, 1, 0]
 ];
 
 const frameDefaults: F.FrameNode = {
-  type: "FRAME",
-  name: "_",
-  x: 0,
-  y: 0,
-  width: 0,
-  height: 0,
-  // New props so annoying
-  backgrounds: [],
-  blendMode: "PASS_THROUGH",
-  locked: false,
-  visible: true,
-  layoutMode: "NONE",
-  primaryAxisSizingMode: "FIXED",
-  counterAxisSizingMode: "FIXED",
-  primaryAxisAlignItems: "MIN",
-  counterAxisAlignItems: "MIN",
-  paddingLeft: 0,
-  paddingRight: 0,
-  paddingTop: 0,
-  paddingBottom: 0,
-  itemSpacing: 0,
-  horizontalPadding: 0,
-  verticalPadding: 0,
-  layoutGrids: [],
-  gridStyleId: "",
-  clipsContent: false,
-  guides: [],
   id: "",
+  name: "Test Frame",
   removed: false,
+  visible: true,
+  locked: false,
+  componentPropertyReferences: null,
+  opacity: 1,
+  blendMode: "PASS_THROUGH",
+  isMask: false,
+  effects: [],
+  effectStyleId: "",
+  relativeTransform: [
+    [1, 0, 1134],
+    [0, 1, -488]
+  ],
+  x: 1134,
+  y: -488,
+  width: 375,
+  height: 812,
+  rotation: 0,
+  layoutAlign: "INHERIT",
+  constrainProportions: false,
+  layoutGrow: 0,
+  layoutPositioning: "AUTO",
   children: [],
-  expanded: false,
-  backgroundStyleId: "",
-  strokeCap: "NONE",
-  strokeMiterLimit: 0,
+  exportSettings: [],
+  fills: [
+    {
+      type: "SOLID",
+      visible: true,
+      opacity: 1,
+      blendMode: "NORMAL",
+      color: {
+        r: 1,
+        g: 1,
+        b: 1
+      }
+    }
+  ],
+  fillStyleId: "",
   strokes: [],
   strokeStyleId: "",
-  strokeWeight: 0,
-  strokeJoin: "ROUND",
-  strokeAlign: "CENTER",
+  strokeWeight: 1,
+  strokeAlign: "INSIDE",
+  strokeJoin: "MITER",
   dashPattern: [],
+  strokeCap: "NONE",
+  strokeMiterLimit: 4,
+  fillGeometry: [
+    {
+      windingRule: "NONZERO",
+      data: "M0 0L375 0L375 812L0 812L0 0Z"
```

Comment:
this is based on running the plugin right?

Reply:
Yes :) Just making sure that this test matches reality and e.g. doesn't include keys we don't allow.